src/main/resources/config/application.yml

```diff
@@ -64,7 +64,7 @@ artemis:
                 java:
                     # possible overrides: maven, gradle
                     default: "ls1tum/artemis-maven-template:java17-18"
-                    maven_blackbox: "ghcr.io/uni-passau-artemis/artemis-dejagnu:20"
+                    maven_blackbox: "ghcr.io/uni-passau-artemis/artemis-dejagnu-ph2:20"
```

Comment:
Note: At the moment, we cannot use the regular image since that would break testing in #6736.
Before this is merged we should update the default image in https://github.com/uni-passau-artemis/artemis-ci-images accordingly to include the new pipeline helper version and revert back to the previous image name here.

In case additional changes are requested for #6736 and an additional round of reviews is required anyway, we could also merge the changes made here into the other PR.